ntc_templates/templates/zyxel_os_cfg_wlan_get.textfsm

```diff
@@ -0,0 +1,20 @@
+Value INDEX (\d+)
+Value BAND (2\.4GHz|5GHz)
+Value SSID (\S+?)
+Value ENABLED (0|1)
+Value BANDWIDTH ((\d+(M|G|)Hz)?)
```

Comment:
Similar to others, if this doesn't exist in some cases, need to have tests and likely an alternative strategy.